examples/kotlin-multiplatform/build.gradle

```diff
@@ -23,10 +21,14 @@ kotlin {
         compilations.create("builtInExecutor") { associateWith(compilations.main) }
     }
     wasm('wasmJs') { d8() }
-    if (HostManager.host == KonanTarget.MACOS_X64.INSTANCE) macosX64('native')
-    if (HostManager.host == KonanTarget.MACOS_ARM64.INSTANCE) macosArm64('native')
-    if (HostManager.hostIsLinux) linuxX64('native')
-    if (HostManager.hostIsMingw) mingwX64('native')
+
+    // Native targets
+    macosX64()
+    macosArm64()
+    linuxX64()
+    mingwX64()
+
+    applyDefaultHierarchyTemplate()
```

Comment:
I believe we should strive that the project contains no custom `dependsOn` so that the template is applied automatically.